wbooks/src/app/components/bookCommentList/index.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+import { FlatList, ListRenderItem, View, Text, TouchableOpacity } from 'react-native';
+import { CommentBook as ICommentBook } from '@interfaces/commentBookInterface';
+
+import BookComment from '../bookComment';
+
+import styles from './styles';
+
+const MINELEMENT = 2;
+const MAXELEMENT = 10;
+
+function BookCommentList({ commentList }: any) {
+  const [quantityElement, setQuantityElement] = useState(MINELEMENT);
+  let commentListFilter: any[] = [];
+
+  const filterComments = () => {
+    return commentList.slice(0, quantityElement);
+  };
```

Comment:
```suggestion
  const filterComments = () => commentList.slice(0, quantityElement);
```

Reply:
The declaration form of the arrow function was changed